offload/liboffload/API/CMakeLists.txt

```diff
@@ -2,8 +2,8 @@
 # include directory. These files are checked in with the rest of the source,
 # therefore it is only needed when making changes to the API.
 
-find_program(CLANG_FORMAT clang-format PATHS ${LLVM_TOOLS_BINARY_DIR} NO_DEFAULT_PATH)
-if (CLANG_FORMAT)
+find_program(clang_format clang-format PATHS ${LLVM_TOOLS_BINARY_DIR} NO_DEFAULT_PATH)
```

Comment:
find_program creates an entry into the `CMakeCache.txt`. By convention, those are upper case.